### DISCUSSION TITLE
Adjust left scrollbar in tailwind ui template

### DISCUSSION THREAD

**Discussion by ignutanz** [tailwindlabs/tailwindcss#9228] [General]
I am using https://demos.creative-tim.com/soft-ui-dashboard-tailwind/pages/dashboard.html?_ga=2.192775594.148648844.1661927141-305318585.1661147278 
I want the scrollbar on the left to fit screen height .How do I do it?

**Comment by smithliam** [comment]
Add `h-full` class to `div.max-h-screen`
